torchbenchmark/models/doctr/__init__.py

```diff
@@ -0,0 +1,34 @@
+from doctr.models import ocr_predictor
+import numpy as np
+import torch
+
+# TorchBench imports
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+from typing import Tuple
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.DETECTION
+    DEFAULT_EVAL_BSIZE = 255
+
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+
+        self.model = ocr_predictor(det_arch='db_resnet50', reco_arch='crnn_vgg16_bn', pretrained=True).to(self.device)
+        # fake document file
+        input_page = (self.batch_size * np.random.rand(600, 800, 3)).astype(np.uint8)
```

Comment:
Is there a way to use torch.Tensor here?

Reply:
You can load the model and input like below with torch.tensor:

## Model
```
model = ocr_predictor(reco_arch = "crnn_vgg16_bn", pretrained=True)
model.reco_predictor.model = model.reco_predictor.model.eval()
```

## Random input
```
input = torch.randn(1, 3, 32, 128)
input = input.type(torch.FloatTensor).to("cuda")
model = model.type(torch.FloatTensor).to("cuda")
```

Reply:
The ocr_predictor wont work like that. You will need to benchmark both models separately because the ocr_predictor is two models stitched together with postprocessing in between. 

The best way to benchmark would be to extract the reco_predictor and det_predictor as two separate models and pass tensors through each like elvin has shown. 

recognition model expects bs, 3, 32, 128
detection model expects bs, 3, 1024, 1024 shape inputs

Reply:
Thanks, separated to two models.